File: Python/basic/test03.py
name = input('이름을 입력: ')
print(name, '님 반갑습니다.')

kor = int(input('국어점수: '))
print('국어점수: ', kor, '점')

avg = float(input('평균: '))
print('평균:', avg, '점')

print("{}님의 국어점수는 {}점이고 평균은{}점입니다.".format(name, kor, avg))
print("%s님의 국어점수는 %d점이고 평균은 %.2f점입니다."%(name, kor, avg))

File: Java/day02/src/Test03.java
import java.util.Scanner;

public class Test03 {

	public static void main(String[] args) {
		System.out.print("이름을 입력 : ");
		Scanner scan = new Scanner(System.in);
//		String name = scan.next();	// 문자열처리, 공백, 탭, 엔터가 나오면 끝남
		String name = scan.nextLine();	// 엔터가 나오면 끝남
//		System.out.println(name + "님 반갑습니다.");
		System.out.print(name + "님의 국어점수 : ");
		int kor = scan.nextInt();
		System.out.println(name + "님의 국어점수는" + kor  + "점입니다.");
		
		scan.close();
	}

}
